src/dataLayer/mongo/user.js

```diff
@@ -66,3 +66,24 @@ export async function createUser(root, vars, ctx) {
     return exists[0];
   }
 }
+
+export async function deleteUser(root, vars, ctx) {
+  const { decoded } = verifyWebToken(ctx);
+  const { accountLinkId } = vars;
+  const loggedInId = decoded[namespace + 'accountLinkId'];
+
+  const exists = await asyncErrorHandler(
+    doesExist(UserModel, { accountLinkId })
+  );
+  if (isEmpty(exists)) {
```

Comment:
I believe this should throw instead. I'd suggest including the `accountLinkId` used too. Something like so:

```js
    throw new Error('There is no account with: accountLinkId ' + accountLinkId);
```

Throwing like this will return a 200 -fine for GraphQL, and add the message to the  "errors" array if it exist.